src/k-provider/ott/provider.js

```diff
@@ -104,7 +104,7 @@ export default class OTTProvider extends BaseProvider<OTTProviderMediaInfoObject
       }
       if (data.has(OTTAssetLoader.id)) {
         const assetLoader = data.get(OTTAssetLoader.id);
-        if (assetLoader && assetLoader.response) {
+        if (assetLoader && assetLoader.response && Object.keys(assetLoader.response).length) {
```

Comment:
when will this happen? why do we check this?

Reply:
Shouldn't happened if the response is valid